examples/cmd/pool.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/src-d/metadata-retrieval/github"
+)
+
+type DownloadersPool struct {
+	Size    int
+	pool    chan *DownloaderPoolItem
+	started bool
+	ended   bool
+	t0      time.Time
+	t1      time.Time
+	stats0  map[int]*downloaderStats
+	stats1  map[int]*downloaderStats
```

Comment:
it looks like `stats1` isn't accessible after calling `End` once. So maybe we don't need to save it into the struct then. The same goes for `t1`.